.make/test.mk

```diff
@@ -471,6 +471,7 @@ $(eval ALL_PKGS_COMMA_SEPARATED := $(6))
 $(eval COV_OUT_FILE := $(COV_DIR)/$(PACKAGE_NAME)/coverage.$(TEST_NAME).mode-$(COVERAGE_MODE))
 @$(ENV_VAR) F8_DEVELOPER_MODE_ENABLED=1 F8_POSTGRES_HOST=$(F8_POSTGRES_HOST) F8_LOG_LEVEL=$(F8_LOG_LEVEL) \
 	go test  $(PACKAGE_NAME) \
+		$(GO_TEST_BINARIES_PARALLEL_FLAG) \
```

Comment:
This should only be applied when we're running integration tests. The only reason for why this already looks so complicated is that we need to take special steps to calculate coverage accross packages. Let's not make this more complicated then it already is.

Reply:
Can you send a PR, please? I thought this might be the correct place to add this flag.